src/main/java/io/fabric8/elasticsearch/plugin/ConfigurationSettings.java

```diff
@@ -87,6 +87,12 @@
      * Kibana index.
      */
     static final String OPENSHIFT_DYNAMIC_ENABLED_FLAG = "openshift.acl.dynamic.enabled";
+    
+    /**
+     * The strategy to use for generating roles and role mappings
+     */
+    static final String OPENSHIFT_ACL_ROLE_STRATEGY = "openshift.acl.role_strategy";
```

Comment:
I added a default to the config but I prefer not to advertise we have this option.